src/pages/en/tutorial/1-setup/1.md

```diff
@@ -0,0 +1,174 @@
+layout: ~/layouts/TutorialLayout.astro
+title: Create a new project
+setup: |
+  import Badge from '~/components/Badge.astro';
+  import Checklist from '~/components/Checklist.astro';
+  import Goals from '~/components/tutorial/Goals.astro';
+---
+
+<Goals>
+  - run the `create-astro` setup wizard to install an Astro project in a local or cloud workspace
+  - started the Astro server in development (dev) mode
+  - viewed a live preview of your website in your browser
+  - made your first edit to your new website!
+</Goals>
+
+## Prepare your Development Environment
+
+This tutorial will include instructions for two types of development environments: ***local*** and ***cloud***.
+
+### 🖥️ Developing locally
+
+If you are **developing locally**, using files and applications that exist on your own computer, then your development environment includes **the terminal application** for your computer's operating system and your **code editor**. In this tutorial, we will use VS Code as the code editor. You will use a web browser to preview your work as you build.
+
+You will have a local copy of your files, and your repository on GitHub will be a copy kept in sync with your local project.
+
+### 🌐 Developing in the cloud
+
+If you are **developing in the cloud**, using StackBlitz or CodeSandbox, then your development environment is your **project workspace running in that browser tab**, which will provide a terminal console and an online version of VS Code in your browser. You will also have a simple browser pane to preview your work as you build. 
+
+There will be no local copy of your files, but there will be a working copy stored in your online account, which you will keep in sync with your source repository on GitHub.
+
+:::tip[working in the cloud?]
+There's no local development to set up! Your project will automatically run in development mode and display a website preview when you open the "Completely Empty" template from https://astro.new in StackBlitz or CodeSandbox. 
+
+**Next Steps**: 
+1. Click the button to "fork" the template (save to your own account dashboard).
+2. Skip ahead in this tutorial to [view a preview of your site](#view-a-preview-of-your-website).
+:::
+
+## Run the `create-astro` setup wizard
+
+The preferred way to create a new Astro site is through our `create-astro` setup wizard.
+
+1. In the command line of your terminal, run the command:
+`npm create astro@latest`
+2. Confirm `y` to install `create-astro`
+3. When the prompt asks, "Where would you like to create your app?" type in the name of a folder to create a new directory for your project:
+`./astro-site`
+
+    :::note
+    A new Astro project can only be created in a completely empty folder, so choose a name for your folder that does not already exist!
+    :::
+
+3. You will see a short list of starter templates to choose from. Use the arrow keys (up and down) to navigate to the "Completely Empty" template, and then press return (enter) to submit your choice. 
```

Comment:
I just checked astro.new and it's still "Completely empty" there (not "empty project)." I can't easily run create astro myself right now, so I'm relying on you. Assuming that's the case, maybe a PR to astro.new so that the template names match? (I'm happy to do that if you agree. Just ping me so I know.) Until that happens, I'll keep using "Completely empty" in the StackBlitz instructions, but will update if we make the names consistent!